src/components/CommonUserModal.tsx

```diff
@@ -145,8 +161,8 @@ const AddLearnerModal: React.FC<AddLearnerModalProps> = ({
       tenantCohortRoleMapping: [
         {
           tenantId: tenantId,
-          roleId: RoleId.STUDENT,
-          cohortId: [selectedCenterCode],
+          roleId:userType==="STUDENT"? RoleId.STUDENT: userType==="TEACHER"?RoleId.TEACHER:RoleId.TEAM_LEADER,
```

Comment:
@AkshataKatwal16 get the types from the app.constant files

Reply:
done